data/tmpl/utility/content1.html

```diff
@@ -8,7 +8,7 @@
   <body>
     <section class="sidebar-content utility">
       <header>
-        <img src="/data/media/headers/utility/content1.gif" width="320" height="220" alt="Curious Fox" />
+        <img src="/data/media/headers/utility/content1.gif" max-width="320" max-height="220" alt="Curious Fox" />
```

Comment:
`max-width/height` is not a valid HTML attribute, only in CSS. As these images are scaled down to 280x193px, I reckon we can just set the `width` and `height` to those values explicitly.